### PR TITLE
feat: add resources to existing tutorials

### PR DESCRIPTION
**Pull request by terichadbourne**
@fsdiogo Can you tell what I'm doing wrong here? It's obviously all dummy data still but I thought I'd done all the steps that would make the resources pages show up. It shows up in the list of lessons on the landing page but if you click to go to it the entire page is blank.

No useful clues in the console or terminal. I already tried `npm install` for good measure.

### CODE REVIEW

**Comment by terichadbourne** [comment]
@vmx @mikeal Do you have recommendations for resources that I can link to from either of these tutorials? Both are presented as being about IPFS but use the DAG API so have an IPLD connection. (So I'll include DAG API docs, what else?) Articles, videos, demos, etc. all welcome.
- [P2P data links with content addressing ](http://localhost:3000/#/basics/)
- [Blogging on the Decentralized Web](http://localhost:3000/#/blog/)

**Comment by terichadbourne** [comment]
@mikeal I've also found lots on content addressing and general "why decentralized web" for the ["decentralized data structures" tutorial](http://localhost:3000/#/data-structures/) but nothing I like for Merkle DAG. Suggestions on that front?

**Comment by terichadbourne** [comment]
@alanshaw @lidel @olizilla Have any favorite already-published resources on MFS to recommend besides these ones? (Looking for other learning materials to link to from the MFS tutorial on ProtoSchool. 

image

**Comment by olizilla** [comment]
I think those are the best we have _right now_. I think @alanshaw covered some MFS in Core Course A from camp. If there is a video from that, it would be my top pick for things to refer people to.

As a github tip, you can embed previews of code from a PR in comments by viewing the specific file you've changed (hidden behind the dot-dot-dots in the top right of each file change in the PR) and shift+click on the line numbers to highlight a selection... grab the URL and drop it in a comment like so:

https://github.com/ProtoSchool/protoschool.github.io/blob/7696f8e68dd55bbeec62971384eb40673aba2456/src/static/tutorials.json#L68-L72

see: https://help.github.com/en/articles/creating-a-permanent-link-to-a-code-snippet

**Comment by terichadbourne** [comment]
Thx for the feedback and tip @olizilla. I'll check in with Zac offline to see when/where the video of @alanshaw & @mikeal's course will be posted and add that when it's ready. That one would also be useful for the Merkle DAG section of the Decentralized Data Structures tutorial.

**Comment by fsdiogo** [comment]
> @fsdiogo Can you tell what I'm doing wrong here?

You were missing the `/resources` at the end of the URL, in the `main.js` file. I've committed a fix, it's working now!

**Comment by mikeal** [comment]
I’ll second the use of the materials from the course we did, that’s the best material I’ve ever seen on creating DAG’s, @alanshaw did an amazing job.

Some of it I’m planning on adapting into some IPLD documentation. I’ll let ya’ll know when that is available and maybe you can link to that as well, but I don’t have a great timeline for it yet.

**Comment by terichadbourne** [comment]
This CSS issue is making the new resources pages look bad, but is actually an overarching issue of the footer not sticking to the bottom of the page: https://github.com/ProtoSchool/protoschool.github.io/issues/255

This is ready for an official review. I think I'm as I good as I can get on the resources lists by myself. Very open to feedback on changing what's included!

**Comment by fsdiogo** [comment]
> This CSS issue is making the new resources pages look bad, but is actually an overarching issue of the footer not sticking to the bottom of the page: #255

I've fixed that issue! Merge PR https://github.com/ProtoSchool/protoschool.github.io/pull/259, then update this branch and you can see the final result.

**Comment by terichadbourne** [comment]
closes #263

File: src/components/Resources.vue [hunk 1]
```diff
@@ -1,9 +1,11 @@
 <template>
   <div class="lesson-text lh-copy">
+    <p>Ready to learn more? There are plenty of additional resources to explore, both in ProtoSchool and beyond. <span v-if="data.length > 1">Here are some of our favorites:</span> <span v-else>Here's our top pick:</span></p>
     <div v-for="(item, idx) in data" :key="`resources-${idx}`" class="mb3">
       <p class="ma0 flex items-center">
         <a class="b blue link" :href="item.link" target="_blank">{{item.title}}</a>
-        <span class="ml2 ph1 bg-navy br-pill white f7">{{item.type}}</span>
+        <span class="ml2 ph2 bg-navy br-pill white f7">{{item.type}}</span>
+        <span v-if="item.link.includes('/proto.school/')" class="ml2 ph2 bg-aqua br-pill navy f7">ProtoSchool</span>
```
Comment (fsdiogo) [inline src/components/Resources.vue]: I'd probably make the text white for consistency, but it's fine either way.

**Comment by terichadbourne** [comment]
@fsdiogo I believe I've successfully merged in your CSS fix and gotten it to work with my contextual footer-picking (one to collect feedback on full tutorial from resources page, the old one to collect individual feedback on each lesson). Mind taking another look from your end to ensure it works for you? 

Also addressed the inconsistent color.

File: src/static/tutorials.json [hunk 2]
```diff
@@ -9,6 +9,49 @@
...
+				"description": "Use the IPFS DAG API to create create verifiable links between dataset with Content Identifiers (CIDs)."
+			},
+			{
+				"title": "Decentralized Web Primer: The Power of Content-Addressing",
+				"link": "https://flyingzumwalt.gitbooks.io/decentralized-web-primer/content/avenues-for-access/lessons/power-of-content-addressing.html",
+				"type": "article",
+				"description": "A beginner-friendly primer on content addressing from Matt Zumwalt."
+			},
+ 			{
+				"title": "The Next Internet Revolution (Excerpt)",
```
Comment (fsdiogo) [inline src/static/tutorials.json]: I'd remove the `(Excerpt)` as it's in the description.

File: src/static/tutorials.json [hunk 3]
```diff
@@ -19,7 +62,18 @@
 			{ "to": "/basics/01", "name": "Create a node and return a Content Identifier (CID)" },
 			{ "to": "/basics/02", "name": "Create a new node that's linked to an old one" },
 			{ "to": "/basics/03", "name": "Read nested data using link" }
-		]
+		],
+	  "resources": [
+	    { "title": "JS-IPFS DAG API Documentation",
```
Comment (fsdiogo) [inline src/static/tutorials.json]: Do we need the `Documentation` here when we have the tag `docs`?

File: src/static/tutorials.json [hunk 4]
```diff
@@ -33,7 +87,13 @@
 			{ "to": "/blog/05", "name": "Add a new tag linked to multiple blog posts" },
 			{ "to": "/blog/06", "name": "List posts chronologically with a chain of links" },
 			{ "to": "/blog/07", "name": "Traverse through all posts, starting with the most recent" }
-		]
+		],
+	  "resources": [
+	    { "title": "JS-IPFS DAG API Documentation",
```
Comment (fsdiogo) [inline src/static/tutorials.json]: Do we need the `Documentation` here when we have the tag `docs`?

File: src/static/tutorials.json [hunk 5]
```diff
@@ -51,6 +111,34 @@
 			{ "to": "/mutable-file-system/09", "name": "Copy a file or directory" },
 			{ "to": "/mutable-file-system/10", "name": "Read the contents of a file" },
 			{ "to": "/mutable-file-system/11", "name": "Remove a file or directory" }
+		],
+	  "resources": [
+	    {
+				"title": "JS-IPFS Files API Documentation",
```
Comment (fsdiogo) [inline src/static/tutorials.json]: Do we need the `Documentation` here when we have the tag `docs`?

**Comment by terichadbourne** [comment]
Thanks for the feedback, folks. I'm merging this with the resources I tracked down, but am open to continued suggestions for improving these lists, and will add content from IPFS Camp when it becomes available.

**Pull request merged**